src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -102,11 +104,14 @@ internal OpenTelemetryLoggerProvider(OpenTelemetryLoggerOptions options, IServic
                     configurationActions[i](serviceProvider, this);
                 }
 
+                OpenTelemetrySdkEventSource.Log.OpenTelemetryLoggerProviderEvent($"Number of actions configured = {configurationActions.Count}.");
```

Comment:
Allocation here is probably not a big deal because this is only called once. @reyang thoughts?

Reply:
+1, I think any Provider/Processor/Exporter ctor, static methods, configuration methods are low frequency use so allocation or something like LINQ would be totally okay.

Reply:
So i will leave it as it is.